src/app/legacy/psammead-assets/src/svgs/navigationIcons.jsx

```diff
@@ -4,7 +4,7 @@ import styled from '@emotion/styled';
 const svgSize = '2.75rem'; // 44px
 
 const MenuIcon = styled.svg`
-  color: #fff;
+  color: #000000;
   fill: currentColor;
```

Comment:
Could be an SVG bug here similar to this: https://github.com/bbc/simorgh/issues/10051
Not sure if it applies to this svg, but it might be a good idea to test it anyway.